### PR TITLE
Craft 5 Support

### PR DESCRIPTION
**Pull request by marknotton-yello**
Updated composer to allow for Craft 5 installations.
Optimised the icon SVG so as to be more suitable if used as raw markup.
Added icon reference for the field.

### CODE REVIEW

**Comment by wbrowar** [comment]
Hi @marknotton-yello, thanks for making these changes and for optimizing the SVG. I was planning on working on the Craft 5 update this weekend and I'll start by taking a look at your PR!

File: src/icon.svg
Comment (wbrowar) [inline src/icon.svg:1]: Ah, I appreciate this! I remember when I made this icon it was sort of quick and I meant to go back and tweak it.

**Comment by wbrowar** [comment]
Thanks, @marknotton-yello! I merged this in especially for the icon stuff. If you wanted to test this out, I think you could install this package as `"wbrowar/craft-little-layout": "dev-main"`. I'll likely change the version number and make a few more tweaks, then I want to wait until Craft 5 is officially released before tagging the final release.

**Comment by wbrowar** [comment]
@marknotton-yello, Little Layout is up on the Craft 5 Plugin Store. I'm going to test this out on a few of my own Craft projects before tagging a full release. Here's the link if you want to take a look: https://plugins.craftcms.com/little-layout?craft5

PS. I still haven't gotten to the Quick Field issue on the Carft 4 branch, but I'd still like to come back to it now that this release it out.

File: src/icon.svg
Comment (marknotton-yello) [reply src/icon.svg:1]: You're welcome! Optimising the SVG removes class name references that are used when exporting SVG's from Illustrator. This prevents conflicts if another SVG on the page uses the same class name, avoiding broken styles.

**Comment by marknotton-yello** [comment]
Thanks for the quick review. We're planning to upgrade many of our projects to Craft 5 and often use Little Layout—it's a very useful plugin! So far, the tests on your latest release haven't shown any issues 🙂

**Comment by marknotton-yello** [comment]
> PS. I still haven't gotten to the Quick Field issue on the Carft 4 branch, but I'd still like to come back to it now that this release it out.

Thanks for the update. I really wouldn't sweat over it. It's not causing me any real-world issues and I'm likely the only one to have experienced this problem. If it's a pain to debug, just close it 👍

**Pull request merged**